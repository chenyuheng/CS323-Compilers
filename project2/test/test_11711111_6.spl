int t;

int main() {
    t = get_div(1, 0);
}

int get_div(float a, float b) {
    return a/b;
}
