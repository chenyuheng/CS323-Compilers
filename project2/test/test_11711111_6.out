Error type 9 at Line 4: invalid argument for function, type(s) not compatible
Error type 5 at Line 4: unmatching type on both sides of assignment
Error type 8 at Line 8: incompatiable return type
